Content.Server/Administration/Logs/AdminLogManager.Json.cs

```diff
@@ -26,61 +26,101 @@ private void InitializeJson()
         foreach (var converter in _reflection.FindTypesWithAttribute<AdminLogConverterAttribute>())
         {
             var instance = _typeFactory.CreateInstance<JsonConverter>(converter);
-            (instance as IAdminLogConverter)?.Init(_dependencies);
             _jsonOptions.Converters.Add(instance);
         }
 
         var converterNames = _jsonOptions.Converters.Select(converter => converter.GetType().Name);
         _sawmill.Debug($"Admin log converters found: {string.Join(" ", converterNames)}");
     }
 
-    private (JsonDocument Json, HashSet<Guid> Players, List<AdminLogEntity> Entities) ToJson(
+    private (JsonDocument Json, List<AdminLogPlayer> Players, List<AdminLogEntity> Entities) ToJson(
+        int id,
         Dictionary<string, object?> properties)
     {
-        var entities = new Dictionary<EntityUid, AdminLogEntity>();
-        var players = new HashSet<Guid>();
-        var parsed = new Dictionary<string, object?>();
+        var entities = new List<AdminLogEntity>();
```

Comment:
If we don't want entities to be saved anymore, the first step would be to not add them before performing the database change at a later date. Not adding them shouldn't break anything nor require a migration, it will just mean that servers won't add to the table as they update with this change.